src/pyproject.toml

```diff
@@ -28,6 +28,7 @@ warn_return_any = true
 [tool.pylint.messages_control]
 disable = [
   "consider-using-f-string",
+  "duplicate-code",
```

Comment:
We already had a number of in-code suppressors of this `pylint` complaint, and I think that a growing collection of component drivers are going to have graph tasks that look "duplicate" from `pylint`'s perspective, so it might be less annoying to suppress at this level. I suspect that "we're all adults here" and will DRY our code whenever and wherever we can.